### PR TITLE
Add recurrence tests

### LINKED ISSUE #113: add tests for recur, add and edit of recurrence

### PR DESCRIPTION
**Pull request by umergta**
fixes #113 

tests added so far
- Recurrence
- ModelManager
- ModuleBook
- UniqueTaskList
- JsonAdaptedTask
- Task
- RecurCommand
- RecurCommandParser

### CODE REVIEW

File: src/test/java/seedu/module/model/ModuleManagerTest.java [hunk 1]
```diff
@@ -43,28 +43,28 @@ public void moduleIsValid_methodTest() {
 
     @Test
     public void insertingNullModuleIntoMappingWithTask_throwsNullPointerException() {
-        Task editedAlice = new TaskBuilder(QUIZ)
```
Comment (yhtMinceraft1010X) [inline src/test/java/seedu/module/model/ModuleManagerTest.java:46]: Nice!

**Review by yhtMinceraft1010X — COMMENTED** [review]
LGTM

**Review by yhtMinceraft1010X — COMMENTED** [review]
LGTM

File: src/test/java/seedu/module/model/task/RecurrenceTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    public void constructor_validRecurrenceString_success() {
+        //EP: valid recurrence string
+        String validRecurrenceString = "daily";
+        RecurrenceType recurrenceType = RecurrenceType.valueOf("daily");
+        assertNotEquals(null, new Recurrence(validRecurrenceString).value);
+        assertEquals(recurrenceType, new Recurrence(validRecurrenceString).getRecurrenceType());
+    }
+
+    @Test
+    public void isValidRecurrenceTest() {
```
Comment (QY-H00) [inline src/test/java/seedu/module/model/task/RecurrenceTest.java:33]: Nice test for different cases!

**Review by QY-H00 — COMMENTED** [review]
Overall LGTM!

**Pull request merged**